docs/site/pages/dsl/index.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: DSL Content Overview
+---
+
+# TSX/JSX Content Authoring (Player DSL)
+
+While Player content _can_ be written directly in JSON, it's definitely not the preferable authoring format. To take advantage of existing developer tool-chains, Player provides a mechanism for authoring content in (J/T)SX as React components and simple TypeScript objects. The Player CLI can then be used to transpile the React tree into a JSON content.
```

Comment:
I think the intro sentence could go into selling the DSL a little more, such as linting, validation, brevity, etc. Maybe even showing a basic comparison of JSON versus DSL, showing the auto id, auto collection wrapping, auto text wrapping?

Reply:
Yeah I can write some copy for that